components/forms/w-form.vue

```diff
@@ -1,27 +1,30 @@
 <script setup>
-  import { watchEffect } from 'vue'
-  import { absentProp } from '#util'
-  import { modelProps } from 'create-v-model'
-  import { createValidationCollector } from './validation'
-
-  const props = defineProps({
-    as: {
-      default: 'form'
-    },
-    shouldValidate: {
-      type: null,
-      default: absentProp
-    },
-    ...modelProps({ modelDefault: absentProp }),
-    ...modelProps({ modelName: 'completed', modelDefault: absentProp }),
-  })
-
-  const emit = defineEmits(['update:modelValue']);
-  const { allChildrenValid, completed, childrenShouldValidate } = createValidationCollector()
-
-  if (props.modelValue !== absentProp) watchEffect(() => emit('update:modelValue', allChildrenValid.value))
-  if (props.completed !== absentProp) watchEffect(() => emit('update:completed', completed.value))
-  if (props.shouldValidate !== absentProp) watchEffect(() => childrenShouldValidate.value = props.shouldValidate)
+import { watchEffect } from 'vue';
+
+import { modelProps } from 'create-v-model';
+
+import { createValidationCollector } from './validation';
+
+import { absentProp } from '#util';
```

Comment:
A good example of why I would have preferred to not have the blank line in between import groups... 😄 😜

Reply:
Yeah but I think it would be difficult to change again in the `@warp-ds/eslint-config` so that it doesn't make a blank line for specific use-cases, while keeping a blank line for other use-cases where we have more imports in each import groups. Either we decide to have a blank line or no blank line. :)